frontend-react/src/components/USLink.tsx

```diff
@@ -123,13 +122,12 @@ export const USLinkButton = ({
         },
         className,
     );
-    return (
-        <SafeLink
-            {...anchorHTMLAttributes}
-            className={linkClassname}
-            extLinkIcon={extLinkIcon}
-        />
-    );
+    if (isExternalUrl(sanitizeHref(anchorHTMLAttributes.href))) {
```

Comment:
Moved the "is a link external" and removed the `extLinkIcon` since Joseph's awesome new `USExtLink` component does all that for us. We had duplicate logic here and I removed it.